src/biokbase/narrative/app_util.py

```diff
@@ -14,10 +14,11 @@
 app_version_tags = ["release", "beta", "dev"]
 
 
-def check_tag(tag, raise_exception: bool = False):
+def check_tag(tag: str, *, raise_exception: bool = False) -> bool:
```

Comment:
Force the typing to treat `raise_exception` as a kwarg instead of positional. I don't know why ruff has a hard time with that.